tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "declaration": false,
+        "allowJs": true,
+        "allowSyntheticDefaultImports": true,
+        "jsx": "preserve",
+        "lib": ["dom", "es2017"],
+        "module": "esnext",
+        "target": "esnext",
+        "moduleResolution": "node",
+        "noEmit": true,
+        "noUnusedLocals": true,
```

Comment:
If this is true, then is the defined ESLint rule still needed?

Reply:
Probably not. Still not totally sure about the overlapping responsibilities of ESLint and TS. I guess initially we can leave all the rules enabled, but in the long run we can probably make ESLint faster by disabling the rules that are already caught by type-checking